src/singletons/Toasts.cpp

```diff
@@ -35,6 +48,38 @@ bool Toasts::isEnabled()
     return false;
 }
 
+QString Toasts::findStringFromReaction(const ToastReactions &reaction)
+{
+    auto iterator = Toasts::reactionToString.find(reaction);
+    if (iterator != Toasts::reactionToString.end())
+    {
+        return iterator->second;
+    }
+    else
+    {
+        return DONT_OPEN;
+    }
+}
+
+QString Toasts::findStringFromReaction(
+    const pajlada::Settings::Setting<int> &value)
+{
+    int i = static_cast<int>(value);
+    return Toasts::findStringFromReaction(static_cast<ToastReactions>(i));
+}
+
+ToastReactions Toasts::findReactionFromString(const QString &string)
```

Comment:
I just realised this is probably not used anymore, because we connected the proper signal to the combobox now. I want to delete the unused before merging. Will check and commit that later!